staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission_test.go

```diff
@@ -0,0 +1,649 @@
+/*
```

Comment:
The tests is exactly the same as those in the `validating/admission_test.go`. I'm not sure if we should keep this file.